config/signatures-test.txt

```diff
@@ -1 +1,2 @@
 com.puppycrawl.tools.checkstyle.AbstractModuleTestSupport#verify(com.puppycrawl.tools.checkstyle.api.Configuration, java.lang.String, java.lang.String[]) @ Use inline config parser instead.
+com.puppycrawl.tools.checkstyle.PropertyCacheFileTest#createTempFile(java.lang.String, java.lang.String, java.io.File) @ Use of this method is forbidden, please use @TempDir for better temporary directory control
```

Comment:
`java.nio.file.Files.createTempDirectory(java.lang.String)  @ Use of this method is forbidden, please use @TempDir for better temporary directory control`